source/PokerSetup.py

```diff
@@ -0,0 +1,344 @@
+#   Code still needs to incorporate a pot.
+
+from source.CardGames import *
+import sys
+
+PlayersStillInGameList = PlayerList
+BlindsList = PlayersStillInGameList
+PlayersPlayingRoundList = PlayersStillInGameList
+PlayersOutOfGameList = []
+
+class PokerGameSetup:
+
+    def __init__(self, name, money: int = 0):
+        self.name = []
+        self.hand = []
+        self.money = money
+
+    def preGame(self, players: PlayerList):
+        #   Give each player their money.
+        #
+        #   Option to have each player start with the same amount of money,
+        #   or with various amounts of money.
+        try:
+            Same = input('''Enter "Same" if each player starts with the same amount.
+Enter "Different" if each player start with different amounts: ''')
+
+            #   Each player starts with the same amount.
+            if Same == "Same" or Same == "same" or Same == "SAME" or Same == "S" or Same == "s":
+                Amount = int(input('No cents! Enter amount: $'))
+                for player in PlayerList:
+                    Player.addMoney(self, amount)
+
+            #   Each player starts with different amounts.
+            elif Same == "Different" or Same == "different" or Same == "DIFFERENT" or Same == "D" or Same == "d":
+                for player in PlayerList:
+                    Amount = int(input('No cents! Enter amount: $'))
+                    Player.addMoney(self, amount)
+
+            #   If a word other than "Same" or "Different" is entered.
+            else:
+                print('''YOU WERE SUPPOSE TO ENTER "Same" OR "Different"!!!
```

Comment:
I like this nice calm way of conveying errors XD

Reply:
Lol, I wanted to think up as many scenarios as I could to break it. I could have done a loop that ran 3-4 times then kill it, but I decided to kill it immediately instead.